docs/spec/v1beta1/receiver.md

```diff
@@ -229,6 +230,23 @@ spec:
       name: webapp
 ```
 
+### Nexus receiver
+
+```yaml
+apiVersion: notification.toolkit.fluxcd.io/v1beta1
+kind: Receiver
+metadata:
+  name: nexus-receiver
+  namespace: default
+spec:
+  type: nexus
+  secretRef:
+    name: webhook-token
+  resources:
+    - kind: ImageRepository
+      name: webapp
+```
```

Comment:
Please add the note here like we do for Harbor and mention the header we use `X-Nexus-Webhook-Signature`